### ISSUE TITLE
After turning on "Do not keep activities" in the system settings, the preference information will not be retained

### ISSUE THREAD

**Issue by sjl872964789** [jroal/a2dpvolume#294]
**Describe the bug**
After turning on "Do not keep activities" in the system settings, the preference information will not be retained.

**Context**
 - Device: [Plxel xl]
 - OS version: [Android 10.0]
 - App version: [2.13.0.4]

**How to reproduce**
1. Turn on "Do not keep activities" in the developer options of the system settings
2. Install and open a2dpvolume
3. Click the three dots in the upper right corner
4. Click on "Preferences"
5. Turn on "Respond to Audio Jack"
6. back
7.Preferences does not take effect

**Expected behavior**
Preferences can take effect

**Screenshots**
The device on the right has the "Do not keep activities" option turned on, while the device on the left does not
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/15941953/90259802-33d38580-de7d-11ea-9af0-4e695446e35e.gif)

**Comment by jroal** [comment]
I don't think this is really worth fixing.  It is very low priority in my opinion.  I am not actively developing this app anymore, just very basic maintenance.  If somebody want to find a fix and post that here, I may get around to implementing it someday.

**Comment by tingsu** [comment]
Thanks for your reply and comments, @jroal.  I totally understand you are maintaining this app in your leisure time for non-profit. This app indeed lasts for a long time for our users. Thanks for your and others' continuous efforts :) 

I just hope to share some inputs. Hope it could help. I also observed this issue when I used this app before without enabling the option "Do not keep activities".  The key point of using "Do not keep activities" is to simulate some adverse system behaviors (e.g., the system may kill some activities when the phone's memory is low). In this regard, I think this issue is somehow critical as it disrupts the app functionality when the phone is low in resources.
